tools/wtv/.shed.yml

```diff
@@ -0,0 +1,15 @@
+name: wtv
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/wtv"
+homepage_url: "https://github.com/RECETOX/wtv"
+categories:
+  - Metabolomics
+description: "A cli/python library for ion selection in mass spectrometry."
+long_description: "WTV is a command-line and Python library designed for ion selection in mass spectrometry workflows. "
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "{{ tool_name }} tool from the wtv package"
+suite:
+  name: suite_wtv
+  description: tools from the wtv suite are used for ion selection
+  type: repository_suite_definition
```

Comment:
Given this is only a single tool, we don't have to define a suite but a single tool will be sufficient - you can see the .shed.yml file of the isolib tool for an example.